packages/nuxt/src/core/nuxt.ts

```diff
@@ -557,6 +557,12 @@ async function initNuxt (nuxt: Nuxt) {
     addPlugin(resolve(nuxt.options.appDir, 'plugins/payload.client'))
   }
 
+  // Show compatibility version banner when Nuxt is running with a compatibility version
+  // that is different from the current major version
+  if (!(satisfies(nuxt._version, nuxt.options.future.compatibilityVersion + '.x'))) {
+    console.info(`running with compatibility version \`${nuxt.options.future.compatibilityVersion}\``)
```

Comment:
Some "Learn more" link would be sweet - though tough to generalize

Reply:
Agreed. I think we can skip though, as users will have explicitly opted in to `compatibilityVersion` in order for banner to show, so I think they will know where to look.